lib/shopify_api/resources/assigned_fulfillment_order.rb

```diff
@@ -0,0 +1,24 @@
+module ShopifyAPI
+  class AssignedFulfillmentOrder < Base
+    CANCELLATION_REQUESTED = 'cancellation_requested'
+    FULFILLMENT_REQUESTED = 'fulfillment_requested'
+    FULFILLMENT_ACCEPTED = 'fulfillment_accepted'
+
+    ALL_ASSIGNED_STATUSES = [
+      CANCELLATION_REQUESTED = 'cancellation_requested',
+      FULFILLMENT_REQUESTED = 'fulfillment_requested',
+      FULFILLMENT_ACCEPTED = 'fulfillment_accepted'
+    ].freeze
+
+    def self.all(options = {})
+      params = options[:params] || options['params'] || {}
+      assigned_status = params[:assigned_status] || params['assigned_status']
```

Comment:
Do we need to be matching on both string and symbol keys for the options hash?

Reply:
There's no guarantee what users' custom code will pass in as a hash. The existing code is mostly :symbol with just a few stray 'keys', so we can just use :symbol here.